docs/contracts.rst

```diff
@@ -428,8 +428,20 @@ change by overriding).
 Constant State Variables
 ************************
 
-State variables can be declared as constant (this is not yet implemented
-for array and struct types and not possible for mapping types).
+State variables can be declared as constant. In this case, they have to be
```

Comment:
Maybe the ``constant`` here can be quoted.

In other parts of the documentation, the English word "constant" is used for something else.
```
``bool``: The possible values are constants ``true`` and ``false``.
```